src/contact/reducers/Reducer.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import * as ActionTypes from '../actions/ActionTypes';
+
+const INITIAL_STATE = {
+	details : null,
```

Comment:
Why you haven't denormalised the store, Any change in these big objects lead to re-render in entire components where they are mapped.

Reply:
Please explain this?